modules/learn/pages/clusters-and-availability/rebalance-and-index-service.adoc

```diff
@@ -0,0 +1,681 @@
+* Shard Based Rebalance and Rebalance Based on File Transfer are synonyms for File-Based Rebalance.
+====
+
+For examples of Index Rebalance and FBR in action, see <<#fbr-all-examples,Understanding Index Rebalance with Examples>>.
+
+[#shard-affinity-and-fbr]
+==== Relationship Between Shard Affinity and FBR
+
+The Shard Affinity is related to FBR (or Shard Based Rebalance) in the following ways:
+
+* As partitions with the same shard ID reside in the same shard file, a rebalance can relocate those partitions by copying that file to another node.
+
+* Without shard affinity, partitions of the same logical shard may be spread across multiple files or nodes.
+This distribution makes FBR inefficient forcing it to use the slower DCP rebuild method.
+For this reason, a single configuration setting enables both shard affinity and FBR.
+
+[#enabling-fbr]
+==== Enabling FBR
+
+For self-managed clusters, FBR is disabled by default.
+You need to enable FBR to use it.
+
+* To enable FBR from the UI, do the following:
+
+.. On your cluster, in the xref:manage:manage-settings/general-settings.adoc#index-storage-mode[Index Storage Mode], enable Standard Global Secondary mode.
++
+For more information about Standard Global Secondary index storage mode, see  xref:learn:services-and-indexes/indexes/storage-modes.adoc#standard-index-storage[Standard Index Storage].
+
+.. Select *Enable File Transfer Based Rebalance*.
+
+* To enable FBR using the REST API, set the `enableShardAffinity` parameter in xref:rest-api:post-settings-indexes.adoc[].
+
+NOTE: <<#smart-batching,Smart Batching>> does not work if you have enabled FBR on your cluster.
+
+For Couchbase Capella (Provisioned Clusters), FBR is enabled by default.
+It can be disabled through a support request.
+The on and off behavior is the same as in self-managed clusters.
+
+==== Restarting an FBR
+
+If an FBR fails, you can start a new rebalance. 
+The second rebalance does not re-transfer any index that the failed rebalance finished moving. 
+Instead, it moves just the indexes that had not moved or were in the process of moving when the error occurred.
+
+To restart FBR, enable it.
+For more information, see <<#enabling-fbr, Enabling FBR>>.
+
+
+// Remove after the 7.x series.
+ifeval::['{page-component-version}' == '7.6'] 
+NOTE: In Couchbase Server versions 7.6.0 and 7.6.1, when you enabled FBR you could not choose which Index Service nodes would contain an index when using the `CREATE INDEX` statement.
+In Couchbase Server 7.6.2 and later, you can use the `WITH <node>` clause to set which node contains the index.
+You still cannot use the `WITH <node>` clause with `ALTER INDEX` after you enable FBR.
+endif::[] 
+
+[#swap-rebalance-for-index-service]
+=== Swap Rebalance for Index Service
+
+A swap rebalance removes one or more source nodes and adds the same number of destination nodes.
+During the swap rebalance operation, indexes move only between those source and destination nodes.
```

Comment:
As mentioned in the other review comment, it is better to mention what happens on cancelled / failed rebalance + retry.